app/frontend/src/features/search/SearchResultsList.tsx

```diff
@@ -22,19 +22,28 @@ const useStyles = makeStyles((theme) => ({
   mapResults: {
     height: "15rem",
     zIndex: 3,
-    overflow: "visible",
-    padding: theme.spacing(3),
+    overflowY: "auto",
+    backgroundColor: theme.palette.background.default,
     [theme.breakpoints.up("md")]: {
       height: "auto",
-      overflow: "auto",
+      width: "30rem",
+      padding: theme.spacing(3),
     },
   },
   baseMargin: { margin: theme.spacing(2) },
   searchDesktop: {
+    "& > div > *": {
+      flexShrink: 0,
+      width: "100%",
+    },
   },
   scroller: {
     marginTop: theme.spacing(3),
-    "&&": { alignItems: "flex-start" },
+    alignItems: "flex-start",
```

Comment:
Sometimes the && is needed to increase the specificity to override the Material-UI style. At the moment this line doesn't get applied (but it doesn't matter for vertical scrolling)